packages/shared/modals/EditAnnotations.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { k8sPatch } from '@openshift-console/dynamic-plugin-sdk';
+import { K8sKind } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+import * as _ from 'lodash-es';
```

Comment:
```suggestion
import * as _ from 'lodash';
```
not sure the reason for this, but IIC I got this suggestion on one of my PR as well...